nixos/modules/image/repart-verity-store.nix

```diff
@@ -194,6 +193,24 @@ in
                 chmod +w ${config.image.repart.imageFileBasename}.raw
               '';
 
+              # replace "TBD" with the original roothash values
+              preInstall = ''
```

Comment:
Shouldn't the second derivation already return a repart-output.json that contains the final values?

Reply:
The output from the intermediate image includes the hashes
```
<snip>
  "activity" : "create",
  "roothash" : "7c915215f4..............."
}
</snip>
```
but in the final image there is just the literal "TBD" string. I don't think repart tries to retrieve and match hashes for verity siblings again when the partitions are already pre-populated.
Output from the build of `finalImage` looks like this:
```
<snip>
  "activity" : "unchanged",
  "roothash" : "TBD"
}
</snip>
```